wallet/db.c

```diff
@@ -477,6 +477,7 @@ static struct migration dbmigrations[] = {
 	 ");"), NULL},
     {SQL("ALTER TABLE channels ADD shutdown_scriptpubkey_local BLOB;"),
 	 NULL},
+    {SQL("ALTER TABLE channels ADD COLUMN our_funding_satoshi INTEGER;"), NULL},
```

Comment:
Should we initialize it to one of the other fields?  Or is the null value a special flag indicating a v1 channel?  (If so, a comment here would be great).